laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -32,6 +32,22 @@ object eq {
   }
 
   /**
+   * Create an approximation of Eq[Eq[A]] by generating 100 values for A
+   * and comparing the application of the two eqv functions
+   */
+  implicit def catsLawsEqForEq[A](implicit arbA: Arbitrary[(A, A)], booleanEq: Eq[Boolean]): Eq[Eq[A]] = new Eq[Eq[A]] {
```

Comment:
I don't really think we need the `Eq[Boolean]` here. I think we assume that primitives are okay to compare directly.